input/metadata/properties/default.json

```
@@ -4745,6 +4810,20 @@
           "description": "Fraction of residue that will decompose in a day given optimal conditions.\nUnitless.\nReference SWAT Input: .BSN - RSDCO.",
           "minimum": 0.0,
           "default": 0.05
+        },
+        "active_N_percent": {
+          "type": "number",
+          "description": "",
+          "minimum": 0.0,
+          "maximum": 100.0,
+          "default": 0.0
+        },
+        "denitrification_rate": {
+          "type": "number",
+          "description": "",
+          "minimum": 0.0,
+          "maximum": 1.0,
+          "default": 0.0
```

```suggestion
        },
```
These should be removed from inputs, instead of being allowed in.

---

Ok I will remove these in the metadata and input json files.